src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -225,7 +225,7 @@ private function addLongOption($name, $value)
 
         $option = $this->definition->getOption($name);
 
-        if (null !== $value && !$option->acceptValue()) {
+        if (null !== $value && !$option->acceptValue() && 'verbose' !== $name) {
```

Comment:
Even if it solves this issue, i assume i don't like that.

`ArgvInput` & `ArrayInput` shouldn't be aware of `Application` configuration (like this `verbose` option name).
Or can we assume these objects are bound to live together ?